src/governance-vote.cpp

```diff
@@ -11,59 +11,52 @@
 
 std::string CGovernanceVoting::ConvertOutcomeToString(vote_outcome_enum_t nOutcome)
 {
-    switch (nOutcome) {
-    case VOTE_OUTCOME_NONE:
-        return "NONE";
-        break;
-    case VOTE_OUTCOME_YES:
-        return "YES";
-        break;
-    case VOTE_OUTCOME_NO:
-        return "NO";
-        break;
-    case VOTE_OUTCOME_ABSTAIN:
-        return "ABSTAIN";
-        break;
+    static const std::map<vote_signal_enum_t, std::string> mapOutcomeString = {
+        { VOTE_OUTCOME_NONE, "none" },
+        { VOTE_OUTCOME_YES, "yes" },
+        { VOTE_OUTCOME_NO, "no" },
+        { VOTE_OUTCOME_ABSTAIN, "abstain" } };
+
+    const auto& it = mapOutcomeString.find(nOutcome);
+    if (it == mapOutcomeString.end()) {
+        LogPrintf("CGovernanceVoting::%s -- ERROR: Unknown outcome %s\n", __func__, nOutcome);
```

Comment:
I'm not familiar with formatting, can it handle `nOutcome` as `%s`?

Reply:
Not sure ping: @UdjinM6 @nmarley

Reply:
I wouldn't experiment with this and would simply use %d :)